### PR TITLE
Wrong attempts is restricted to 5 times

### PR DESCRIPTION
**Pull request by chandanvarma009**

### CODE REVIEW

File: Yona/Yona/Login/LoginViewController.swift [hunk 1]
```diff
@@ -85,17 +99,49 @@ extension LoginViewController: KeyboardProtocol {
 
 extension LoginViewController: CodeInputViewDelegate {
     func codeInputView(codeInputView: CodeInputView, didFinishWithCode code: String) {
+//        if loginAttempts == totalAttempts {
```
Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: Always delete commented out code

File: Yona/Yona/Login/LoginViewController.swift [hunk 2]
```diff
@@ -85,17 +99,49 @@ extension LoginViewController: KeyboardProtocol {
...
                 navigationController?.pushViewController(dashboardStoryboard, animated: true)
             }
         } else {
             errorLabel.hidden = false
             codeInputView.clear()
+            if loginAttempts == totalAttempts {
+                let defaults = NSUserDefaults.standardUserDefaults()
+                defaults.setBool(true, forKey: YonaConstants.nsUserDefaultsKeys.isBlocked)
+                defaults.synchronize()
+                self.displayAlertMessage("Login", alertDescription: "Your account has been locked for security reasons, use the reset passcode option to enable our account.")
```
Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: We can't put strings into the code they have to all be in the LOcalizable file

File: Yona/Yona/Login/LoginViewController.swift (hunk 1 above)
Comment (chandanvarma009) [inline Yona/Yona/Login/LoginViewController.swift]: Sorry I forgot.

File: Yona/Yona/Login/LoginViewController.swift (hunk 2 above)
Comment (chandanvarma009) [inline Yona/Yona/Login/LoginViewController.swift]: I agree, I am waiting for the text message to be given by Yona team.BTW I will replace it. Thanks for pointing.

Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: Yes, but for now, can you put the string in the localizable file

File: Yona/Yona/Login/LoginViewController.swift [hunk 3]
```diff
@@ -85,17 +99,49 @@ extension LoginViewController: KeyboardProtocol {
 
 extension LoginViewController: CodeInputViewDelegate {
     func codeInputView(codeInputView: CodeInputView, didFinishWithCode code: String) {
+//        if loginAttempts == totalAttempts {
+//            let defaults = NSUserDefaults.standardUserDefaults()
```
Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: remove commewnted out code

File: Yona/Yona/Login/LoginViewController.swift [hunk 4]
```diff
@@ -44,6 +47,17 @@ class LoginViewController: UIViewController {
             codeInputView!.becomeFirstResponder()
         }
         
+        if let attempts = NSUserDefaults.standardUserDefaults().boolForKey(YonaConstants.nsUserDefaultsKeys.isBlocked) as? Bool {
+            if attempts  {
+                self.displayAlertMessage("Login", alertDescription: "Your account has been locked for security reasons, use the reset passcode option to enable our account.")
+                codeInputView!.resignFirstResponder()
+                codeInputView!.userInteractionEnabled = false
+                errorLabel.hidden = false
+                errorLabel.text = "Your account has been locked for security reasons, use the reset passcode option to enable our account."
```
Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: this needs to be put in localisable

File: Yona/Yona/Login/LoginViewController.swift [hunk 5]
```diff
@@ -44,6 +47,17 @@ class LoginViewController: UIViewController {
             codeInputView!.becomeFirstResponder()
         }
         
+        if let attempts = NSUserDefaults.standardUserDefaults().boolForKey(YonaConstants.nsUserDefaultsKeys.isBlocked) as? Bool {
+            if attempts  {
+                self.displayAlertMessage("Login", alertDescription: "Your account has been locked for security reasons, use the reset passcode option to enable our account.")
```
Comment (benSmith1981) [inline Yona/Yona/Login/LoginViewController.swift]: localizable string replace the text here with one please

**Pull request merged**